src/generate-analysis.ts

```diff
@@ -124,11 +126,12 @@ function buildAnalysis(members: Members, packagePath: string): Analysis {
     namespace.functions.push(serializeFunction(_function, packagePath));
   }
 
+  // TODO(usergenic): Consider moving framework-specific code to separate file.
   for (const behavior of members.polymerBehaviors) {
     const namespaceName = getNamespaceName(behavior.name);
     const namespace = namespaces.get(namespaceName) || analysis;
-    namespace.mixins = namespace.mixins || [];
-    namespace.mixins.push(
+    setDefaults(namespace, {metadata: {polymer: {behaviors: []}}});
```

Comment:
I tend to prefer wordy explicit code when reading this sort of thing, as it's unclear how such functions handle edge cases (is undefined overwritten? is null?). If we need to use it several times then the abstraction pays more rent. If `metadata` was typed I'd also object on those grounds too.

All that said though, this is a fairly minor objection as there aren't any nulls or undefineds here, tests will cover us pretty well, and metadata is untyped, so nonblocking.